dataplane/loader/src/main.rs

```diff
@@ -20,6 +21,8 @@ use log::{info, warn};
 struct Opt {
     #[clap(short, long, default_value = "lo")]
     iface: String,
+    #[clap(flatten)]
+    tls_config: GrpcConfig,
```

Comment:
maybe?

```suggestion
    grpc_tls_config: GrpcConfig,
```

Reply:
Good call out.

I have changed the name of the struct to match the function argument: `GrpcConfig` --> `TlsConfig`. There's nothing about the config that makes it unique to Grpc so I don't know why I named it that way originally.